pkgs/applications/editors/vim/plugins/overrides.nix

```
@@ -471,20 +471,9 @@ self: super: {
     dependencies = with self; [ lush-nvim ];
   });
 
-  himalaya-vim = buildVimPluginFrom2Nix {
-    pname = "himalaya-vim";
-    inherit (himalaya) src version;
+  himalaya-vim = super.himalaya-vim.overrideAttrs (old: {
     dependencies = with self; [ himalaya ];
```

I tried it out and ran into this:
```
E605: Exception not caught: Himalaya CLI not found, see https://github.com/soywod/himalaya#installation
```
```suggestion
    postPatch = ''
      substituteInPlace plugin/himalaya.vim \
        --replace "if !executable('himalaya')" "if v:false"
      substituteInPlace autoload/himalaya/request.vim \
        --replace "'himalaya" "'${pkgs.himalaya}/bin/himalaya"
    '';
```